src/writer.js

```diff
@@ -10,64 +10,86 @@ define([
     util
 ) {
     var createXForm = function (form) {
```

Comment:
I have commented out the implementation of `createXForm` and tried to move the functionality to `createXform.js`